javacutil/src/main/java/org/checkerframework/javacutil/TreeUtils.java

```diff
@@ -2607,36 +2607,7 @@ public static boolean isCallToVarArgsMethodWithZeroVarargsActuals(MethodInvocati
    * @return true if the given method invocation is a varargs invocation
    */
   public static boolean isVarArgs(NewClassTree newClassTree) {
-    return isVarArgs(elementFromUse(newClassTree), newClassTree.getArguments());
-  }
-
-  /**
-   * Returns true if a method/constructor invocation is a varargs invocation.
-   *
-   * @param method the method or constructor
-   * @param args the arguments passed at the invocation
-   * @return true if the given method/constructor invocation is a varargs invocation
-   */
-  private static boolean isVarArgs(ExecutableElement method, List<? extends ExpressionTree> args) {
-    if (!method.isVarArgs()) {
-      return false;
-    }
-
-    List<? extends VariableElement> parameters = method.getParameters();
-    if (parameters.size() != args.size()) {
-      return true;
-    }
-
-    TypeMirror lastArgType = typeOf(args.get(args.size() - 1));
-    if (lastArgType.getKind() == TypeKind.NULL) {
-      return false;
-    }
-    if (lastArgType.getKind() != TypeKind.ARRAY) {
-      return true;
-    }
-
-    TypeMirror varargsParamType = parameters.get(parameters.size() - 1).asType();
-    return TypesUtils.getArrayDepth(varargsParamType) != TypesUtils.getArrayDepth(lastArgType);
```

Comment:
This part is wrong if the method call has a method type variable.  `method` does not have the type variable substituted, so `varargsParamType` could be a type variable when it should be an array.

For example:
```
Stream<byte[]> stream = Stream.of(new byte[][] {});
```
The above is not a vararg call, this method thinks it is.
`varargsParamType` parameter type is `T[]`, and `lastArgType` is `byte[][]`. So the method returns true.